roles/osasinfra/templates/clientpod.yml.j2

```diff
@@ -0,0 +1,103 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ cifmw_osasinfra_client_pod_name }}
+  namespace: openstack
+spec:
+  containers:
+  - args:
+    - infinity
+    command:
+    - /bin/sleep
+    env:
+    - name: CONFIG_HASH
+      value: n6h58h584h689h588hfdh687h8dh64dh685h5c5h5dfh578h648h5c6h576h54fh559h65bh5bh674h59bh678h655h5ddh55hb7h7hb4h5bch69h668q
+    - name: OS_CLOUD
+      value: default
+    image: {{ cifmw_osasinfra_client_pod_image }}
+    imagePullPolicy: Always
+    name: {{ cifmw_osasinfra_client_pod_name }}
+    resources: {}
+    securityContext:
+      privileged: true
+    terminationMessagePath: /dev/termination-log
+    terminationMessagePolicy: File
+    volumeMounts:
+    - mountPath: /home/cloud-admin/.original-config/openstack/clouds.yaml
+      name: openstack-config
+      subPath: clouds.yaml
+    - mountPath: /home/cloud-admin/.original-config/openstack/secure.yaml
+      name: openstack-config-secret
+      subPath: secure.yaml
+    - mountPath: /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
+      name: combined-ca-bundle
+      readOnly: true
+      subPath: tls-ca-bundle.pem
+    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+      name: kube-api-access-gsbj6
+      readOnly: true
+    - mountPath: /home/cloud-admin/.original-config/cert/
+      name: openstack-cert
+      readOnly: true
+  dnsPolicy: ClusterFirst
+  enableServiceLinks: true
+  imagePullSecrets:
+  - name: openstackclient-openstackclient-dockercfg-tnkcg
+  preemptionPolicy: PreemptLowerPriority
+  priority: 0
+  restartPolicy: Always
+  schedulerName: default-scheduler
+  securityContext: {}
+  serviceAccount: openstackclient-openstackclient
+  serviceAccountName: openstackclient-openstackclient
+  terminationGracePeriodSeconds: 0
+  tolerations:
+  - effect: NoExecute
+    key: node.kubernetes.io/not-ready
+    operator: Exists
+    tolerationSeconds: 300
+  - effect: NoExecute
+    key: node.kubernetes.io/unreachable
+    operator: Exists
+    tolerationSeconds: 300
+  volumes:
+  - configMap:
+      defaultMode: 420
+      name: openstack-config
+    name: openstack-config
+  - name: openstack-config-secret
+    secret:
+      defaultMode: 420
+      secretName: openstack-config-secret
+  - name: combined-ca-bundle
+    secret:
+      defaultMode: 292
+      secretName: combined-ca-bundle
+  - name: kube-api-access-gsbj6
+    projected:
+      defaultMode: 420
+      sources:
+      - serviceAccountToken:
+          expirationSeconds: 3607
+          path: token
+      - configMap:
+          items:
+          - key: ca.crt
+            path: ca.crt
+          name: kube-root-ca.crt
+      - downwardAPI:
+          items:
+          - fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
+            path: namespace
+      - configMap:
+          items:
+          - key: service-ca.crt
+            path: service-ca.crt
+          name: openshift-service-ca.crt
```

Comment:
We shouldn't mount this. This is mounting the serviceaccount from the underlying OCP.

Reply:
Done